woostarter/patterns/footer.php

```diff
@@ -6,20 +6,27 @@
  * Block Types: core/template-part/footer
  */
 ?>
-<!-- wp:group {"metadata":{"name":"Default Footer"},"align":"full","className":"is-style-default","style":{"spacing":{"padding":{"top":"0","bottom":"0"}}},"layout":{"type":"constrained"}} -->
-<div class="wp-block-group alignfull is-style-default" style="padding-top:0;padding-bottom:0"><!-- wp:spacer {"height":"var:preset|spacing|50"} -->
+<!-- wp:group {"tagName":"footer","metadata":{"name":"Default Footer"},"align":"full","className":"is-style-default","style":{"spacing":{"margin":{"top":"0","bottom":"0"},"padding":{"top":"0","bottom":"0"}}},"layout":{"type":"constrained"}} --><footer class="wp-block-group alignfull is-style-default" style="margin-top:0;margin-bottom:0;padding-top:0;padding-bottom:0"><!-- wp:spacer {"height":"var:preset|spacing|50"} -->
```

Comment:
Changing the `tagName` here means we have a double `footer` tag:

<img width="561" alt="image" src="https://github.com/user-attachments/assets/09e7d47b-457a-4f85-815c-aa2443ebd10e" />

I think Assembler uses the template part for the `footer` tag, so we could remove that from the pattern if we do it the same way.

Reply:
ah good call, the header has the same issue, I'll fix both

Reply:
Fixed in https://github.com/woocommerce/woo-themes/pull/123/commits/8ccafdcbcb3580cad0f3ba047d548c04a77df4dd